scripts/lint.sh

```diff
@@ -84,6 +84,15 @@ function test_require_len_zero {
   fi
 }
 
+function test_require_equal_len {
+  if grep -R -o -P 'require\.Equal\((t, )?.*, len\([^,]*$' .; then
```

Comment:
Would this match `require.Equal(t, len(something), 2)`?

I don't think it does, but I think we want it to.

Can't we just do: `require\.Equal\(.*len\(.*\)`?

(I'm kind of rusty on regex)

Reply:
Good catch, it wouldn't match that. I think this would be better:
```suggestion
  if grep -R -o -P 'require\.Equal\(.*, len\(' .; then
```

Reply:
Don't think that matches, for example, `require.Equal(len(foo), 2)`

Reply:
`require\.Equal\((len\()|(.+? len\()`